docs/io/ascii/index.rst

```diff
@@ -229,6 +229,63 @@ ASCII with no loss of information.
 ..
   EXAMPLE END
 
+Miscellaneous Examples
+----------------------
+Some points of interests pertaining to masking are the ``fill_include_names``
+and ``fill_exclude_names`` attributes of the :class:`~astropy.io.ascii.BaseData` class.
+The original intention for the usage of these attributes are in the ``masks``
+method of the same :class:`~astropy.io.ascii.BaseData` class. In the first step of the
+``masks`` method, the mask is evaluated with values from ``fill_values`` as applied
+to which column using ``fill_include_names`` and ``fill_exclude_names``.
+In the second step, all the replacements of values are done for the appropriate columns.
+As standalone tools, they can be used to serve in isolating columns that are of interests
+while not deleting them outright. We can use the ``fill_include_names`` attribute in include
+only columns with names as indicated. On the contrary but highly similarly, we can use
+the ``fill_exclude_names`` attribute to exclude the columns with the names as indicated.
+One use case is where one has a CSV file with empty values indicating missing values,
+but for a string column one might desire those to empty strings "" to be not masked.
+Or perhaps one has a file that uses the number -99 as the missing data indicator
+for most columns but then -99 occurs as real data in another column. These features
+allow the data to be read in with ``astropy.io.ascii`` correctly without too much effort.
+
+Examples
+^^^^^^^^
+..
+  EXAMPLE START
+  Using the ``fill_include_names`` and ``fill_exclude_names`` attributes for ASCII tables
```

Comment:
these are parameters rather than attributes, please update the usage of the word in this patch.

Reply:
Sure, I have updated the wording from `attribute(s)` to `parameter(s)` in the wording.